interface/src/app/gene/gene-network.tpl.html

```diff
@@ -2,17 +2,28 @@
   <h1 class="page-header">
     Gene Network
   </h1>
-  <autocomplete-search-panel organism="searchCtrl.organism" autocomplete="searchCtrl.autocomplete"
-    ng-if="searchCtrl.autocomplete">
+
+  <m-l-model-selector selected-m-l-model="searchCtrl.selectedMLModel">
```

Comment:
Hmmm good question - I guess the names for these tags are converted from the camelCase names of the component and variables by Angular. So instead of ``mLModelSelector`` and ``selectedMLModel`` we could name them ``mlModelSelector`` and ``selectedMlModel``. I was using the rules for initials in camelCase, but I don't have a strong opinion on this, what do you think?